CONTRIBUTING.md

```diff
@@ -1,3 +1,14 @@
 # Contributing to Flyte
 
-For information related to contributing to Flyte, please check out the [Contributing to Flyte](https://docs.flyte.org/en/latest/community/contribute.html) section of the documentation.
+For information related to contributing to Flyte, please check out the [Contributing to Flyte](https://docs.flyte.org/en/latest/community/contribute/index.html) section of the documentation.
+
+## Recommendation Order (For Beginners)
+* Setup dev environment 
+* Read the following and try to run 10 examples
+    * https://docs.flyte.org/en/latest/user_guide/introduction.html
+    * https://docs.flyte.org/en/latest/flytesnacks/userguide.html
+* Finish reading the [Concepts](https://docs.flyte.org/en/latest/user_guide/concepts/main_concepts/index.html)
+* Finish reading the [Control Plane](https://docs.flyte.org/en/latest/user_guide/concepts/control_plane/index.html)
+* Finish reading the [Component Architecture](https://docs.flyte.org/en/latest/user_guide/concepts/component_architecture/index.html)
+* Choose 2 good first issues from the following and start solving them with the knowledge you have read.
+* Familiar with using DOCKERFILE to push images to localhost for development
```

Comment:
While not the examples follow it today, we try to make ImageSpec first, and trying to avoid Dockerfiles